django_project/localities_osm/utilities.py

```diff
@@ -1,8 +1,50 @@
 __author__ = 'Irwan Fathurrahman <irwan@kartoza.com>'
 __date__ = '04/04/19'
 
-from localities_osm.models.locality import LocalityOSMView
+from localities_osm.models.locality import LocalityOSMView, LocalityOSM
 
 
 def get_all_osm_query():
     return LocalityOSMView.objects.filter(osm_id__isnull=False).exclude(name='')
+
+
+def convert_into_osm_dict(locality):
+    data = locality.repr_dict()
+
+    osm_dict = data['values']
+    osm_dict['name'] = data['name']
+
+    if 'staff' in osm_dict.keys():
+        staff = osm_dict['staff'].split('|')
+        osm_dict['staff_doctors'] = staff[0]
+        osm_dict['staff_nurses'] = staff[1]
+        del osm_dict['staff']
+
+    if 'inpatient_service' in osm_dict.keys():
+        beds = osm_dict['inpatient_service'].split('|')
+        total_bed = int(beds[0]) + int(beds[1])
```

Comment:
i think it the value can be "3|"
so one of bed can be empty
please handle this, or it will give error